src/main/java/org/webshar/hrms/dao/ResourceRepository.java

```diff
@@ -0,0 +1,9 @@
+package org.webshar.hrms.dao;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.webshar.hrms.models.db.Resource;
+
+public interface ResourceRepository extends JpaRepository<Resource, Long>
+{
+
+}
```

Comment:
This needs to be present under org.webshar.hrms.repository package as per naming convention practice